HISTORY.rst

```diff
@@ -0,0 +1,18 @@
+.. :changelog:
+
+History
+-------
+
+upcoming
+------------------
+
+* Brought back ``HISTORY.rst``.
+* Changed ``README.md`` into ``README.rst``.
+* Append change history to readme in ``long_description`` of package.
```

Comment:
These last two are very minor, I don't think they're useful in the changelog since they don't really affect users at the code level in any way